store/actions/authentication.js

```diff
@@ -41,6 +41,36 @@ export const confirmAccount = token => async dispatch => {
   })
 }
 
+export const confirmEmailUpdate = (confirmationToken, accept = true) => async dispatch => {
+  let response = null
+  let success = false
+
+  dispatch({ type: actionTypes.CONFIRM_EMAIL_UPDATE })
+
+  try {
+    const token = Cookies.get('accessToken')
+
+    response = await fetch(`/api/email/${confirmationToken}?response=${accept ? 'accept' : 'reject'}`, {
```

Comment:
We should be using the `apiService` rather than using `fetch` directly.

Reply:
from what I'm seeing apiService does not do authentication yet, which is why I've been holding off.

Reply:
Nope, we don't actually need `withCredentials` for our local requests. That prop is only required if you're sending a CORS requests with Auth headers, but we're making requests to the same domain.

The Authorization header also [gets set automatically for us by the `Page` component](https://github.com/RollForGuild/rollforguild.com/blob/develop/components/Page.js#L94-L96).